### DISCUSSION TITLE
What is the difference between these two folders

### DISCUSSION THREAD

**Discussion by huangJW7** [gitpod-io/openvscode-server#418] [Q&A] [answered]
Hello, I am trying to save and upload this folder when shutting down the openvscode-server. 

And the next time I restart the openvscode-server, download and restore this folder.

So that I can make the webide from standard environment and save everyone's extensions.

This is the folder I mentioned `/home/workspace/.openvscode-server/extensions`

But I also see another folder `/home/.openvscode/extensions`

The first folder includes the extensions where I download in the Marketplace, the second folder seems to be pre-installed extensions.

In the 1.67..0 version, I saw some extensions moved from the pre-installed folder to the user-download folder when I download some extensions from the Marketplace. It makes user-download folder very large.

user-extensions folder, I only download these two extensions
![image](https://user-images.githubusercontent.com/57228223/185778336-fa7fc0f7-09bf-482d-8ef5-d670ef45c394.png)

While the pre-install extension is 
![image](https://user-images.githubusercontent.com/57228223/185778392-2c59cab8-bd8d-4673-a10e-ed093af5cd56.png)

And it won‘t happen in 1.70.1 version. 

Will this happen again? And how should I prevent this behavior.🙏

**Comment by jeanp413** [comment, marked as answer]
Hi @huangJW7, the `vscode.*-1.62.3` extensions you saw in version `1.67.0` is a [bug in openvsx](https://github.com/eclipse/openvsx/issues/446), right now it's mitigated in `>1.69.0` as the logic to download this buggy extensions is disabled in stable releases, but it could happen in insiders releases e.g. [openvscode-server-insiders-v1.71.0](https://github.com/gitpod-io/openvscode-server/releases/tag/openvscode-server-insiders-v1.71.0). we are waiting for the fix to be deployed in openvsx
